### ISSUE TITLE
Keyboard navigation doesn't work in Windows menubar

### ISSUE THREAD

**Issue by tdeck** [r0x0r/pywebview#1834]
On the Windows platform, hitting "alt" will highlight the first menu, but the menubar cannot be navigated with the keyboard. This feature is important for screen reader users. In particular:

- The first menu can't be opened by pressing ENTER or the down arrow
- Other menus can't be selected by pressing the left or right arrows

You can observe this yourself by running examples/menu.py.
